gk/main.c

```diff
@@ -2196,6 +2196,12 @@ gk_proc(void *arg)
 			rx_queue_back, tx_queue_front, lcore,
 			&back_num_pkts, back_icmp_bufs, instance, gk_conf);
 
+		if (iter_count >= scan_iter) {
+			entry_idx = (entry_idx + 1) % gk_conf->flow_ht_size;
+			rte_prefetch_non_temporal(
```

Comment:
Should we have a similar function call to ``prefetch_flow_entry()`` using non temporal prefetch?

Reply:
I don't think so because we only need the beginning of a `struct flow_entry` to check if it's expired. But this is not obvious, so we must add a comment above this prefetch to explain why we don't have a second prefetch.

Reply:
Sure, will do.